README.md

```diff
@@ -56,7 +56,7 @@ both compiled in `x64` mode. See section `x86 compilation` for other architectur
 
 ### Compilation of OpenSSL:
 ```
-C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat” amd64
+"C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat” amd64
```

Comment:
while you're at it, you might want to fix the ending quote, which is currently ” [U+201D](https://codepoints.net/U+201D), not the ASCII double quote (i.e. [U+0022](https://codepoints.net/U+0022))

Reply:
Thanks!